src/lib/modules/client/terminal/xterm-wrapper.ts

```
@@ -67,6 +71,59 @@ export async function createTerminal(options: TerminalOptions): Promise<Terminal
+    }
+    return true;
+  });
+
+  // Clipboard image paste interception
+  if (options.terminalId && options.apiKey) {
+    const pasteTermId = options.terminalId;
+    const pasteApiKey = options.apiKey;
+
+    options.container.addEventListener('paste', async (e: ClipboardEvent) => {
+      try {
+        if (!e.clipboardData) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)